src/book/02-projects/01-nubots/03-build-system.mdx

```diff
@@ -5,3 +5,526 @@ title: Build System
 description: The NUbots build and continuous integration system.
 slug: /projects/nubots/build-system
 ---
+
+## The NUbots Build System
+
+### Overview
+
+The NUbots codebase uses a [CMake](www.cmake.org) build system based on the [NUClear Roles framework](https://github.com/Fastcode/NUClearRoles) and the [Ninja](https://ninja-build.org/) build system.
```

Comment:
CMake link does not work, NUbook will consider it an internal link if it does not have https:// at the beginning.